cicd/pipelines/securing-webhooks-with-event-listeners.adoc

```diff
@@ -0,0 +1,23 @@
+[id="securing-webhooks-with-event-listeners"]
+= Securing webhooks with event listeners
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: securing-webhooks-with-event-listeners
+
+toc::[]
+
+Triggers in {pipelines-title} support both HTTP (insecure) and HTTPS (secure HTTP) connections to the `Eventlistener` resource. With the secure HTTPS connection, you get an end-to-end secure connection within and outside the cluster. Enable the secure HTTPS connection, after creating a namespace, for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace. After that, create a `Trigger` resource and a secured route using the re-encrypted TLS termination.
+
+When you install a {pipelines-title} using the Operator, a `tekton-operator-proxy-webhook` pod always runs in the background. The webhook watches for the labels in the namespace, and the `service.beta.openshift.io/serving-cert-secret-name=<secret_name>` annotation is set on the `EventListener` object, whenever one is found. This, in turn, creates secrets and required certificates.
```

Comment:
```suggestion
Installing {pipelines-title} using the Operator runs a `tekton-operator-proxy-webhook` pod in the background. This webhook watches for the labels in the namespace. When you add the label to the namespace, the webhook sets the `service.beta.openshift.io/serving-cert-secret-name=<secret_name>` annotation on the `EventListener` object. This, in turn, creates secrets and required certificates.
```

Reply:
Is it important to call out installing by using the Operator? Are there other methods of installing that don't run the webhook pod in the background?

Reply:
it will be installed using Operator only without Operator webhook pod won't run